CONTRIBUTING.md

```diff
@@ -11,57 +11,57 @@ Please take a moment to review the following guidelines to ensure a smooth and c
 
 ## How to Contribute
 
-We encourage contributions in the form of bug fixes, feature enhancements, or the addition of new demos. To get started, follow these steps:
+🙌 We're all for contributions, be it bug fixes, feature upgrades, or the introduction of new demos. To get started, follow these steps:
 
-1. Fork the Repository:
-   - Click the "Fork" button at the top of this repository.
-   
-2. Clone Your Fork:
-   - Clone the repository to your local machine using the command:
+1. **Fork the Repository:**
+   - Hit that "Fork" button at the top of this repository.
+
+2. **Clone Your Fork:**
+   - Bring the repository to your local machine with this command:
      ```sh
      git clone https://github.com/your-username/You-Dont-Need-JavaScript.git
      ```
 
-3. Create a New Branch:
-   - Create a new branch for your contribution. Use a descriptive name for your branch, e.g., `feature/my-new-demo`.
+3. **Create a New Branch:**
+   - Forge a new branch for your contribution with a name that paints a picture, e.g., `feature/my-new-demo`.
      ```sh
      git checkout -b feature/my-new-demo
      ```
 
-4. Make Your Changes:
-   - Make your desired changes or additions to the project.
+4. **Make Your Changes:**
+   - Work your magic and make the desired changes or additions to the project.
 
-5. Test Your Changes:
-   - Ensure that your changes are functional and do not introduce any issues.
+5. **Test Your Changes:**
+   - Cast a spell of testing to ensure your changes function correctly without introducing any gremlins.
 
-6. Add your work in the README.md file using our [Style Guide](README.md#style-guide).
+6. **Add Your Work to README:**
+   - Weave your masterpiece into the README.md following our [Style Guide](README.md#style-guide).
 
-6. Commit Your Changes:
-   - Commit your changes with a meaningful commit message.
+6. **Commit Your Changes:**
```

Comment:
Repeated "no 6" fix this and push it again @0Armaan025 .

Reply:
Fixed

Reply:
You removed one point from the CONTRIBUTING.md file i.e add you work in README.md, don't do that just change the number(no) of the points.

Reply:
fixed it @aayush105 , thank you for pointing it out!